changes/6962.fixed

```diff
@@ -0,0 +1 @@
+Fixed a `Validation` error with empty ports in bulk editing.
```

Comment:
```suggestion
Fixed a Validation error with empty ports field in Service Bulk Edit Form.
```

Reply:
RadioProfile also uses NumericArrayField in its forms. This PR likely fixes this issue for it as well. Would we want to capture that in the changelog as well?

`RadioProfileBulkEditForm/RadioProfileForm` -> `allowed_channel_list`